components/fs/nfs/lwip_include/lwipopts.h

```diff
@@ -1,13 +1,6 @@
 /*
- * Copyright 2020, Data61
- * Commonwealth Scientific and Industrial Research Organisation (CSIRO)
- * ABN 41 687 119 230.
- *
- * This software may be distributed and modified according to the terms of
- * the BSD 2-Clause license. Note that NO WARRANTY is provided.
- * See "LICENSE_BSD2.txt" for details.
- *
- * @TAG(DATA61_BSD)
+ * Copyright 2020, Data61, CSIRO
```

Comment:
The previous copyright notice was checked by Data61's lawyers.  I think you need to keep the full name of CSIRO, and the ABN.